### PR TITLE
April News

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### pkgcheck reports now include dependencies
+
+Our [automated checking system for packages](/commcalls/dec2021-automation/) submitted for peer review now has a new section, "*Package Dependencies*". 
+This is intended to help editors and reviewers understand how package dependencies (that is, packages listed in *Imports*, *Depends*, or *Suggests* `DESCRIPTION` fields) are actually used by a package.
+The section summarises total numbers of function calls made to each package, followed by collapsible "details" sections containing numbers of calls made to the individual functions of each packages. 
+Dependencies include base R and all recommended packages, ensuring that this section provides a comprehensive overview of how each package submitted for peer review uses and depends upon the entire R ecosystem.
```
Comment (steffilazerte) [inline content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:52]: 😍  Oh I'm in love! I have got to add pkgcheck to my package release workflows, what a great way for catching heavy dependencies that may not be necessary!

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+
+Want to try it for yourself?
+Install [pkgcheck](https://docs.ropensci.org/pkgcheck/) and then run `pkgcheck::pkgcheck (<package-source-directory>)`.
+
+### R-universe now features package individual pages
```
Comment (steffilazerte) [inline content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:57]: Coming soon?

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,573 @@
...
+Our next sessions are:
+
+* [Tuesday, 03 May 2022 9 AM North American Pacific / 16:00 UTC](/events/coworking-2022-05/) "Why be a reviewer of R packages?" *Hosted by community host François Michonneau and [Steffi LaZerte](/author/steffi-lazerte/)*
+  * Cowork on a project of your choice;
+  * Prepare an R package for rOpenSci software review, work on reviewer comments, or review a package;
+  * Or ask [François Michonneau](https://francoismichonneau.net/) what it's like to review packages!
+  
+* [Tuesday, 07 June 2022 9 AM Australian Western / 1:00 UTC ](/events/coworking-2022-06/) " Tackle something you've always wanted to learn" *Hosted by community host Nick Tierney and [Steffi LaZerte](/author/steffi-lazerte/)*
+  * Take time to tackle something you've always wanted to learn;
+  * Cowork on a project of your choice.
```
Comment (steffilazerte) [inline content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:72]:
```suggestion
* [Tuesday, 07 June 2022 9 AM Australian Western / 1:00 UTC ](/events/coworking-2022-06/) "Tackle something you've always wanted to learn" *Hosted by community host Nick Tierney and [Steffi LaZerte](/author/steffi-lazerte/)*
  * Cowork on a project of your choice; 
  * Take this time to tackle something you've always wanted to learn;
  * Or talk to [Nick Tierney](https://ropensci.org/author/nicholas-tierney/) and discuss what it is you want to learn, and get some tips and resources to get yourself started!
```

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,573 @@
...
+### Next coworking sessions
+
+Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffi LaZerte  and various community hosts. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.
+
+Our next sessions are:
+
+* [Tuesday, 03 May 2022 9 AM North American Pacific / 16:00 UTC](/events/coworking-2022-05/) "Why be a reviewer of R packages?" *Hosted by community host François Michonneau and [Steffi LaZerte](/author/steffi-lazerte/)*
+  * Cowork on a project of your choice;
+  * Prepare an R package for rOpenSci software review, work on reviewer comments, or review a package;
+  * Or ask [François Michonneau](https://francoismichonneau.net/) what it's like to review packages!
```
Comment (steffilazerte) [inline content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:68]: For the newsletter, should the links be complete, or '/author/...'?
```suggestion
  * Or ask [François Michonneau](https://ropensci.org/author/francois-michonneau/) what it's like to review packages!
```

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,573 @@
...
+As the maintainer of an rOpenSci package, feel free to contact us on Slack or email `info@ropensci.org` to get your call for maintainer featured in the next newsletter.
+
+## Package development corner
+
+Some useful tips for R package developers. :eyes:
+
+### Identifying code duplication
+
+Are you worried a codebase you've just inherited doesn't follow the DRY (don't repeat yourself) principle?
+You might find the [dupree package](https://russhyde.github.io/dupree/) by Russ Hyde useful: "The goal of dupree is to identify chunks / blocks of highly duplicated code within a set of R scripts.".
```
Comment (steffilazerte) [inline content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:554]: Definitely something I need to check out!

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle! Hardly any suggestions this time, just a couple updates to the coworking (thanks for getting the details in there for me!) and a couple of "so cool!" comments where I couldn't resist :grin:

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle! Hardly any suggestions this time, just a couple updates to the coworking (thanks for getting the details in there for me!) and a couple of "so cool!" comments where I couldn't resist :grin:

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd (hunk 1 above)
Comment (maelle) [reply content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:52]: :100: , awesome work by @mpadge!

**Comment by maelle** [comment]
Thanks a lot @steffilazerte! :pray: 

Yes the links need to be relative here too.

File: content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd (hunk 2 above)
Comment (maelle) [reply content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd:57]: Yes, to be added by @jeroen :wink:

Comment (jeroen) [reply content/blog/2022-04-21-ropensci-news-digest-april-2022/index.Rmd]: Done now :D

**Pull request merged**